src/main/java/com/arpnetworking/clusteraggregator/models/StatusResponse.java

```diff
@@ -294,6 +294,7 @@ public Builder setSha(final String value) {
         /**
          * {@inheritDoc}
          */
+        @SuppressWarnings("deprecation") // The uid() method is apparently deprecated on UniqueAddress
```

Comment:
This just used in the output of the status page.  If the field is deprecated, lets just drop it from the serialized object.